static/js/index.js

```diff
@@ -18,10 +97,61 @@ burger.addEventListener('click', () => {
     iwq.classList.add('show');
   }
 });
-// ====================================================================================
+// ============================================================================
+// eslint-disable-next-line no-unused-vars
+function detail(c) {
+  parpopup.style.display = 'flex';
+  const update = details.filter((item) => item.category === c);
+  parpopup__popup.innerHTML = `
+  <div class="head">
+  <span class="name">${update[0].title}</span>
+  <span id="close" onclick="closes()">&times;</span>
+</div>
+<ul>
+  <li><a href="">${update[0].list.list1}</a></li>
+  <li><a href="">${update[0].list.list2}</a></li>
+  <li><a href="">${update[0].list.list3}</a></li>
+</ul>
+<div class="popup__responsive">
+  <img src="${update[0].image}" alt="" class="responsive" />
+</div>
+<div class="types">
+  <div class="type">
+    <p>${update[0].desc}</p>
+  </div>
+  <div class="type">
+    <ul>
+      <li>${update[0].lang.lang1}</li>
+      <li>${update[0].lang.lang2}</li>
+      <li>${update[0].lang.lang3}</li>
+      <li>${update[0].lang.lang4}</li>
+    </ul>
+    <hr />
+    <div class="btns">
+      <button>See live</button><button>See source</button>
```

Comment:
Great job with your buttons. However, they are missing a few things which I will mention below.

- [ ] Kindly add the button states i.e hover, focus or clicked to your buttons. Cyrrently they are just static and are not intuitively as button-like.
- [ ] Kindly add some spacing between them to separate them from each other to match the chosen Figma design.
- [ ] Also, kindly add the icons which are on the buttons according to the chosen Figma design but are not on your design.

**This is how I am seeing the design right now:**

![image](https://user-images.githubusercontent.com/86318284/195417906-70d2a6b6-583e-4066-ae12-ef35b99168a3.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/195417959-c162e912-07b7-4247-b859-dcd1ae65357c.png)